src/schema/v2/sale/index.ts

```diff
@@ -172,19 +172,11 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
           }))
         },
       },
-      cascadingEndTimeInterval: {
```

Comment:
I don't think we can remove this until we update all clients to use the other field.

Reply:
Force already uses the other field, and @annacarey has changed Eigen to use the new field.